specification/07-theory_operations.adoc

```diff
@@ -880,6 +879,104 @@ TVM ->> TVM: Use device interface
 
 === Interface Unbinding
 
+As the platform resources owner, only the host supervisor domain manager may
+decide to unbind an interface from a TVM. At any moment, it can choose to do so
+through the `sbi_covh_unbind_interface()` `COVH` ABI. The targeted TSM services
+the request by moving the TDI from the TDISP `CONFIG_LOCKED` or `RUN` state back
```

Comment:
I think it handles `ERROR` state as well.

Reply:
Fixed.